Таск "Як зробити індивідуальне створення URL".

YuliyaGoncharova opened the pull request:
Додала декілька рядків і трохи змінила в класі URLServiceImpl в методі createShortURL. Решта змін - використала код, що міститься в гілці Liza (тобто це не моя робота по факту, а робота Лізи).

Commit:
Таск "Як зробити індивідуальне створення URL".
Додала декілька рядків в клас URLServiceImpl в метод createShortURL.
Решта змін - використала код, що міститься в гілці Liza.

Elliisiv commented:
Код потестила все працює , просто його треба буде додати на новий сервіс щоб показувало того хто робив, а не я скопіювала його в свою гілку.

Commit:
Метод isShortUrlUnique() створено.
Додана перевірка чи є короткий урл унікальним (виклик isShortUrlUnique() в методі createShortURL() ).

Sergiy-Chernuha committed:
Some refactor

Sergiy-Chernuha committed:
Merge remote-tracking branch 'origin/master' into unique_url_yuz

# Conflicts:
#	src/main/java/ua/goit/shortener/url/repositories/URLRepository.java
#	src/main/java/ua/goit/shortener/url/services/impl/URLServiceImpl.java

Sergiy-Chernuha committed:
confirm merge conflict
